scipy/spatial/src/distance_wrap.c

```diff
@@ -32,6 +32,16 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/* avoid unknown pragma warning in clang which may be used by developers */
+#ifndef __clang__
```

Comment:
updated, @rgommers I hope the check you suggested was just to brief for the comment and not actually used in any of your the code, as it will break with the next gcc version 5 due in april.